pkg/staging/glog/glog.go

```diff
@@ -0,0 +1,496 @@
+// Go support for leveled logs, analogous to https://code.google.com/p/google-glog/
+//
+// Copyright 2013 Google Inc. All Rights Reserved.
+// Copyright 2018 The KubeVirt Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This package reimplements github.com/golang/glog and logs to a structured logger.
+package glog
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io"
+	stdLog "log"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync/atomic"
+	"time"
+
+	log2 "github.com/go-kit/kit/log"
+)
+
+type LogLevel int32
+
+const (
+	INFO LogLevel = iota
+	WARNING
+	ERROR
+	FATAL
+)
+
+var LogLevelNames = map[LogLevel]string{
+	INFO:    "info",
+	WARNING: "warning",
+	ERROR:   "error",
+	FATAL:   "fatal",
+}
+
+var logger log2.Logger
+var v int
+var c string
```

Comment:
why not using more meaningful var names?

Reply:
Improved.